Hotfix infiniteList

philippe-cleany opened the pull request.

philippe-cleany committed:
List: change perPage default value

We fetch 10 item perpage as default.
Now we fetch 25 item as default

philippe-cleany committed:
List: remove hasMore props

In List we set hasMore props to the mapsState to props,
but we notifiyed some trouble in production. Now we removed because
we going to change the way that we set this value to the next commit.
We to this in oder to fix infiniteList.

philippe-cleany committed:
infiniteList: resolve bug with hasMore

we infiniteList we notifiyed that the list still load data if data
does not exist. now we change the way that we compute hasMore, we
compute the number of page and load data if nb page is still lower than
the current page.

philippe-cleany committed:
InfiniteList: change the way we defined hasMore

hasmore is compute directely, with the props total and the query result.
Now we defined hasmore directely as a props. we do this in order to have
this props globaly in order to prevent the next feature who will use
infiniteList component.

philippe-cleany committed:
e2e: test fix

We change the list behavior, we have some trouble with the e2e test.
Now we remove test that we don't use anymore and change to the right
value. We do this in order to success all the test.

philippe-cleany committed:
InfiniteList: fix fetch multipages

With InfiniteList we fetch more than one pages with infiniteList.
Now we fix this in component shouldUpdate by checking if the next page
and the current page is not the same. We do this in order to fix the log
error in the console.

philippe-cleany committed:
InfiniteList: InfiniteList cound't find props

When we trying to mount the commonent after couple minute, we have some
trouble with infiniteList who's thrying to find props indefined. Now we
defined infined list if data have finish load. we to this in order to
fix this error and to mount correctly the component if data is exist.

philippe-cleany committed:
util/getFetchedAt: cache desactivated

By default when an list item is not up to date to 10min,
it was removed from the list. but with infiniteList we have some trouble
with this. now desactivate this behavior in order to have infinteList
stable for now, we will change this behavior in nexted version of
infiniteList.

thomas-cleany reviewed (commented):
Looks good for me, just waiting for the tests

philippe-cleany committed:
util/getFetchedAt: make prettier

sebastien-cleany reviewed (approved):
Some tests missing otherwise look's good to me.

philippe-cleany committed:
util/getFetchedAt: make lint & change the test behavior

We change the behavior of getFetchedAt. in order to have the right
result for the test we make some change.

philippe-cleany committed:
example: add more post exmple

We have more post in the example, accordingly to that we have to make
change to in the e2e test in order to test the app correctly.

philippe-cleany committed:
example/comments: add infiniteList

We replace the List view to the InfiniteList view.
We do this in order to test InfiniteList component.
